### ISSUE TITLE
Tests are failing on Julia 1.6

### ISSUE THREAD

**Issue by simonbyrne** [JuliaLang/PackageCompiler.jl#706]
https://github.com/JuliaLang/PackageCompiler.jl/runs/7175755312?check_suite_focus=true#step:4:340
```
InitError(mod=:LLVMExtra_jll, error=ErrorException("Cannot locate artifact 'LLVMExtra' in '/tmp/jl_s8rPQE/packages/LLVMExtra_jll/RuRRc/Artifacts.toml'"))
```

**Comment by KristofferC** [comment]
Yeah, I merged a test that failed on 1.6 at a time when we didn't have 1.6 CI. It is a bit tricky to make this conditional on the julia version though.

**Comment by DilumAluthge** [comment]
Should be fixed by #771
